float64 slip_spring
float64 standing_leg
float64 hop_height
float64 slip_leg
float64 leg_pos_kp
float64 leg_for_kp
float64 leg_pos_kd
float64 leg_for_kd
float64 hip_pos_kp
float64 hip_pos_kd

uint8 main_controller
uint8 spring_type
uint8 force_type
uint8 stance_controller
uint8 hop_type
